src/Microsoft.AspNetCore.Razor/CodeGenerators/CSharpCodeGenerator.cs

```diff
@@ -146,8 +146,7 @@ protected virtual void BuildConstructor(CSharpCodeWriter writer)
             using (writer.BuildConstructor(Context.ClassName))
             {
                 // Any constructor based logic that we need to add?
-            };
-        }
+            }        }
```

Comment:
That is just a line format issue, i'll correct that. The big item was removal of the unnecessary semicolon at the end